llvm/docs/Coroutines.rst

```diff
@@ -2049,6 +2055,22 @@ the coroutine must reach the final suspend point when it get destroyed.
 
 This attribute only works for switched-resume coroutines now.
 
+coro_must_elide
+---------------
+
+When a Call or Invoke instruction is marked with `coro_must_elide`,
+CoroAnnotationElidePass performs heap elision when possible. Note that for
```

Comment:
I think the name `coro_must_elide` is a misnormer. "must elide" sounds as if it would be a compilation error if elision fails. However, this is not the case

Reply:
What about `coro_elide_safe`?

Reply:
love it!

Reply:
Changed. Closing this one.